.github/workflows/push-mimir-build-image.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         uses: docker/setup-buildx-action@v3
 
       - name: Login to Docker Hub
-        uses: docker/login-action@v3
+        uses: docker/login-action@v2
```

Comment:
```suggestion
        # Temporarily downgrade from v3, since it fails to authenticate (might need a Docker Hub personal access token instead of password)
        uses: docker/login-action@v2
```